src/web/collectors/CollectorsList.jsx

```diff
@@ -70,40 +90,54 @@ const CollectorList = createReactClass({
     const { sort } = this.state;
     const field1 = sort(collector1);
     const field2 = sort(collector2);
+
     return (this.state.sortDesc ? naturalSort(field2, field1) : naturalSort(field1, field2));
   },
 
-  _getTableHeaderClassName(field) {
-    return (this.state.sortBy === field ? (this.state.sortDesc ? 'sort-desc' : 'sort-asc') : 'sortable');
+  _getSortingIcon(field) {
+    const { sortBy, sortDesc } = this.state;
+    const name = (sortBy === field ? (sortDesc ? 'sort-amount-up' : 'sort-amount-down') : 'sort');
+
+    return <StyledIcon name={name} fixedWidth $isAlwaysVisible={sortBy === field} />;
```

Comment:
Is the `$` intentional? What does this do/add/benefit?

Reply:
That is intentional and will make it a transient prop, a prop that is only to be consumed by styled-components and should not be passed down to the rendered element. I did that to fix a `React does not recognize the (...) prop on a DOM element.` warning: https://stackoverflow.com/a/62456493.

Reply:
Well that is pretty awesome and a lot cleaner than the previous methods! Thanks for clarifying!!